catalog/solr/catalog-solr-solrclient/src/main/java/ddf/catalog/source/solr/rest/SolrRest.java

```diff
@@ -100,7 +100,9 @@ public void getProperties() {
       Map<String, Object> map =
           gson.fromJson(response, new TypeToken<Map<String, Object>>() {}.getType());
 
-      fieldTypes = (ArrayList<Object>) map.get("fieldTypes");
+      if (map != null) {
+        fieldTypes = (ArrayList<Object>) map.get("fieldTypes");
+      }
     } catch (Exception e) {
       LOGGER.debug("Unable to getProperties from: {}", getSolrCatalogSchemaUrl(), e);
```

Comment:
❗️ This form will not print the exception since a parameter is used. Either concatenate the parameter to the string instead of passing it to 'debug()` or drop the exception if we do not wish to log it.

Reply:
Dropped exception from logging statement